code/.storybook/main.ts

```diff
@@ -1,5 +1,7 @@
 import { join } from 'node:path';
 
+import react from '@vitejs/plugin-react';
```

Comment:
We were missing this?!?! 😕

Reply:
I have no clue how our Storybook was supposed to work without it. Maybe you have a good explanation!